onionshare/__init__.py

```diff
@@ -168,6 +172,15 @@ def main(cwd=None):
         print(e.args[0])
         sys.exit()
 
+    if mode == 'website':
+        # Prepare files to share
+        print(strings._("preparing_website"))
```

Comment:
As above, please make this a hardcoded English string rather than a translatable `strings._()`